tools/test/examples/examples_lib.py

```diff
@@ -382,27 +382,36 @@ def compile_repos(config, toolchains, targets, profiles, verbose, exp_filter, jo
                     summary_string = "%s %s %s" % (name, target, toolchain)
                     logging.info("Compiling %s" % summary_string)
 
-                    build_command = ["mbed-cli", "compile", "-t", toolchain, "-m", target, "-j", str(jobs)] + (['-vv'] if verbose else [])
-                    if profiles:
-                        for profile in profiles:
-                            build_command.extend(["--profile", profile])
-
-                    logging.info("Executing command '%s'..." % " ".join(build_command))
-                    proc = subprocess.Popen(build_command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-
-                    std_out, std_err = proc.communicate()
-                    std_out = std_out.decode()
-                    std_err = std_err.decode()
-                    print ("\n#### STDOUT ####\n%s\n#### STDERR ####\n%s\n#### End of STDOUT/STDERR ####\n" % (std_out,std_err))
-
-                    if proc.returncode:
-                        failures.append(example_summary)
+                    if cmake:
+                        build_command_seq = ["mbed-tools configure -t {} -m {}".format(toolchain,target), "cmake -S . -B CMAKE_BUILD -GNinja", "cmake --build CMAKE_BUILD"]
```

Comment:
Use `cmake -S . -B CMAKE_BUILD -GNinja -DCMAKE_BUILD_TYPE=${PROFILE}` to set the desired build profile, where ${PROFILE} can be one of `develop`, `debug` or `release`.

Reply:
Is the default profile is develop?
the Jenkins CI is not able to handle multiple profiles at moment,  shall leave with one profile for now, and adding rest of profile as an improvement to the CI later on ?

Reply:
Was the following code, which we are deleting in this PR, not supporting profiles? We could modify it to continue to pass the profile, rather than delete it.
```
                        for profile in profiles:
                            build_command.extend(["--profile", profile])
```

Reply:
And yes, the default is `develop`. Same as with the old tools.